csharp/src/Ice/Communicator.cs

```diff
@@ -144,6 +144,17 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
 
         public ToStringMode ToStringMode { get; }
 
+        /// <summary>Returns the Ice version in the form A.B.C, where A indicates the major version, B indicates the
+        /// minor version, and C indicates the patch level.</summary>
+        /// <returns>The Ice version.</returns>
+        public static string StringVersion => "4.0.0-alpha.0"; // "A.B.C", with A=major, B=minor, C=patch
+
+        /// <summary>Returns the Ice version as an integer in the form A.BB.CC, where A indicates the major version,
+        /// BB indicates the minor version, and CC indicates the patch level. For example, for Ice 3.3.1, the returned
+        /// value is 30301.</summary>
+        /// <returns>The Ice version.</returns>
+        public static int IntVersion => 40000; // AABBCC, with AA=major, BB=minor, CC=patch
```

Comment:
Is Communicator really the best place for the Ice version? Why not just a static class called `Version`?